src/controllers/index.js

```diff
@@ -2,6 +2,8 @@ const express = require('express');
 
 const router = express.Router();
 const home = require('./home');
+const game = require('./game');
 
 router.get('/', home.get);
```

Comment:
it's a choice we have to write routers this way ,, 
```javascript
router
.get('/', home.get)
.get('/game', game.get)
.post('/someroute', game.someroute);
```
it's not an issue but it's good to know